dist/icon/icon.css

```diff
@@ -1,3208 +1,3 @@
-}
-
-svg.icon--unionpay-18-colored {
-    height: 18px;
-    width: 30px;
-}
-
-svg.icon--unionpay-24-colored {
-    height: 24px;
-    width: 38px;
-}
-
-svg.icon--unionpay-32-colored {
-    height: 32px;
-    width: 50px;
-}
-
-svg.icon--unlocked-16 {
-    height: 16px;
-    width: 16px;
-}
-
-svg.icon--unlocked-24,
-svg.icon--unselect-all-24 {
-    height: 24px;
-    width: 24px;
-}
-
-svg.icon--upload-16 {
-    height: 16px;
-    width: 16px;
-}
-
-svg.icon--upload-24 {
-    height: 24px;
-    width: 24px;
-}
-
-svg.icon--venmo-12-colored {
-    height: 12px;
-    width: 20px;
-}
-
-svg.icon--venmo-18-colored {
-    height: 18px;
-    width: 30px;
-}
-
-svg.icon--venmo-24-colored {
-    height: 24px;
-    width: 38px;
-}
-
-svg.icon--venmo-32-colored {
-    height: 32px;
-    width: 50px;
-}
-
-svg.icon--verified-condition-16 {
-    height: 16px;
-    width: 16px;
-}
-
-svg.icon--verified-condition-24,
-svg.icon--video-24 {
-    height: 24px;
-    width: 24px;
-}
-
-svg.icon--visa-12-colored {
-    height: 12px;
-    width: 20px;
-}
-
-svg.icon--visa-18-colored {
-    height: 18px;
-    width: 30px;
-}
-
-svg.icon--visa-24-colored {
-    height: 24px;
-    width: 38px;
-}
-
-svg.icon--visa-32-colored {
-    height: 32px;
-    width: 50px;
-}
-
-svg.icon--wallet-24 {
-    height: 24px;
-    width: 24px;
-}
-
-svg.icon--wallet-64 {
-    height: 64px;
-    width: 64px;
-}
-
-svg.icon--wallet-balance-12-colored {
-    height: 12px;
-    width: 20px;
-}
-
-svg.icon--wallet-balance-18-colored {
-    height: 18px;
-    width: 30px;
-}
-
-svg.icon--wallet-balance-24-colored {
-    height: 24px;
-    width: 38px;
-}
-
-svg.icon--wallet-balance-32-colored {
-    height: 32px;
-    width: 50px;
-}
-
-svg.icon--watch-16 {
-    height: 16px;
-    width: 16px;
-}
-
-svg.icon--watch-24,
-svg.icon--whatsapp-24 {
-    height: 24px;
-    width: 24px;
-}
-
-svg.icon--won-16 {
-    height: 16px;
-    width: 16px;
-}
-
-svg.icon--won-24 {
-    height: 24px;
-    width: 24px;
-}
-
-svg.icon--wrench-16 {
-    height: 16px;
-    width: 16px;
-}
-
-svg.icon--wrench-24,
-svg.icon--youtube-24 {
-    height: 24px;
-    width: 24px;
-}
-
-svg.icon--yuan-16 {
-    height: 16px;
-    width: 16px;
-}
-
-svg.icon--yuan-24 {
-    height: 24px;
-    width: 24px;
-}
-
-svg.icon--zloty-16 {
-    height: 16px;
-    width: 16px;
-}
-
-svg.icon--zloty-24 {
-    height: 24px;
-    width: 24px;
-}
-
-svg.icon--zoom-in-16 {
-    height: 16px;
-    width: 16px;
-}
-
-svg.icon--zoom-in-24 {
-    height: 24px;
-    width: 24px;
-}
-
-svg.icon--zoom-out-16 {
-    height: 16px;
-    width: 16px;
-}
-
-svg.icon--zoom-out-24 {
-    height: 24px;
-    width: 24px;
-}
-
-svg.star-rating--0,
-svg.star-rating--0-5,
-svg.star-rating--1,
-svg.star-rating--1-5,
-svg.star-rating--2,
-svg.star-rating--2-5,
-svg.star-rating--3,
-svg.star-rating--3-5,
-svg.star-rating--4,
-svg.star-rating--4-5,
-svg.star-rating--5 {
-    height: 24px;
-    width: 132px;
-}
-
-svg.star-rating--small-0,
```

Comment:
It looks like `star-rating-select` stopped working on Chrome. 

<kbd><img width="524" alt="image" src="https://github.com/eBay/skin/assets/1675667/cb56cbe4-c3c5-4301-b278-74e75079df6f"></kbd>

Reply:
Fixed it! It was a sass issue